app/v2/connectors/V2BaseConnector.scala

```diff
@@ -34,9 +34,12 @@ trait V2BaseConnector extends HttpErrorFunctions {
   protected def movementsPostDeperatureDeclaration(eoriNumber: EORINumber): UrlPath =
     UrlPath.parse(s"$movementsBaseRoute/traders/${eoriNumber.value}/movements/departures/")
 
+  protected def movementsGetDepartureMessage(eoriNumber: EORINumber, departureId: DepartureId, messageId: MessageId): UrlPath =
+    UrlPath.parse(s"$movementsBaseRoute/traders/${eoriNumber.value}/movements/departures/${departureId.value}/messages/${messageId.value}/")
+
```

Comment:
Why isn't this using the servicesConfig object? That would allow you to source movementsBaseRoute from the appconfig.

Reply:
I don't really care where this is, but I keep hearing this argument flip flop both ways - when do we use a config option vs. just doing it in the code because either way, it requires a release (and we're not going to do it on a per-environment basis, which is the only way to update a config variable without a new release).

If we make it us the `servicesConfig` object, then we need to ensure we inject it in all connectors and that it's defined here. It's not an issue, but it's what we'd need to do.